docs/src/user-guide/guides-using-object-storage/clp-usage.md

```diff
@@ -11,22 +11,20 @@ angle brackets (`<>`) with the appropriate values:
 ```bash
 sbin/compress.sh \
   --timestamp-key <timestamp-key> \
-  <prefix>
+  https://<bucket-name>.s3.<region-code>.amazonaws.com/<prefix>
```

Comment:
If i am to update this, maybe I will do something like : 
```suggestion
  <URL>
  
<URL> can either be a path-style URL or virtual-hosted–style URL. For exmaple, a virtual-hosted–style URL will look like `https://<bucket-name>.s3.<region-code>.amazonaws.com/<prefix>`. For more details, refer to some link.

... cont'd of describing each components.

Note mentioning that instead of key-name, the URL given to us can end with a prefix.

```